### DISCUSSION TITLE
FormBuilderDateTimePicker initial text input mode

### DISCUSSION THREAD

**Discussion by nikel-abibiano** [danvick/flutter_form_builder#1323] [Q&A]
If I have a FormBuilderDateTimePicker for a time field, is it possible to open it with Text Input mode by default and allow the user to switch to the dial picker if needed?

![image](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/75383263/ce5bd0f3-31b2-448c-a39c-307ccf8bd40c)

**Comment by ariefsn** [comment]
hi @nikel-abibiano 
you can try to set `timePickerInitialEntryMode` to `TimePickerEntryMode.dial`
